packages/mobile/src/components/summary-table/SummaryTable.tsx

```diff
@@ -153,16 +165,16 @@ export const SummaryTable = ({
 
   const renderMoreOptionsToggle = () => {
     return (
-      <Flex p='xs'>
-        <TextLink
+      <Flex p='xs' alignSelf='flex-start'>
+        <PlainButton
           onPress={onToggleExtraItems}
-          variant='visible'
-          textVariant='body'
-          size='s'
-          strength='strong'
+          iconRight={showExtraItems ? IconCaretUp : IconCaretDown}
+          disabled={disableExtraItemsToggle}
         >
-          {showExtraItems ? messages.hideAdvanced : messages.showAdvanced}
-        </TextLink>
+          {showExtraItems
+            ? hideExtraItemsCopy || messages.hideAdvanced
```

Comment:
how about `??` instead of `||` (style guide)

Reply:
aye!